plugin/src/main/java/io/fabric8/maven/plugin/mojo/develop/DebugMojo.java

```diff
@@ -251,6 +273,23 @@ private boolean enableDebugging(HasMetadata entity, PodTemplateSpec template) {
                         container.setPorts(ports);
                         enabled = true;
                     }
+                    if (debugSuspend) {
+                        // Setting a random value to force pod restart
+                        this.debugSuspendValue = String.valueOf(new Random().nextLong());
+                        KubernetesResourceUtil.removeEnvVar(env, DebugConstants.ENV_VAR_JAVA_DEBUG_SUSPEND);
+                        KubernetesResourceUtil.setEnvVar(env, DebugConstants.ENV_VAR_JAVA_DEBUG_SUSPEND, this.debugSuspendValue);
```

Comment:
Shouldn't `KubernetesResourceUtil.setEnvVar()` overwrite an existing variable if already present ? I would prefer this over an explicit removeEnvVar before, so that its more aligned to common practice. Also, setEnvVar should also ensure that it does not add the same env var twice.